streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -382,27 +424,41 @@ public static void addAvgMaxLatency(final Sensor sensor,
     public static void addInvocationRateAndCount(final Sensor sensor,
                                                  final String group,
                                                  final Map<String, String> tags,
-                                                 final String operation) {
+                                                 final String operation,
+                                                 final String descriptionOfInvocation,
+                                                 final String descriptionOfRate) {
         sensor.add(
             new MetricName(
-                operation + "-rate",
+                operation + TOTAL_SUFFIX,
                 group,
-                "The average number of occurrence of " + operation + " operation per second.",
+                descriptionOfInvocation,
                 tags
             ),
-            new Rate(TimeUnit.SECONDS, new Count())
+            new CumulativeCount()
         );
         sensor.add(
             new MetricName(
-                operation + "-total",
+                operation + RATE_SUFFIX,
                 group,
-                "The total number of occurrence of " + operation + " operations.",
+                descriptionOfRate,
                 tags
             ),
-            new CumulativeCount()
+            new Rate(TimeUnit.SECONDS, new Count())
         );
     }
 
+    public static void addInvocationRateAndCount(final Sensor sensor,
+                                                 final String group,
+                                                 final Map<String, String> tags,
+                                                 final String operation) {
+        addInvocationRateAndCount(sensor,
+                                  group,
+                                  tags,
+                                  operation,
+                                  "The total number of " + operation,
+                                  "The average per-second number of" + operation);
```

Comment:
nit: missing space after 'of'

Reply:
```suggestion
                                  "The average per-second number of " + operation);
```

@ableegoldman , Good eye! Nit nit: feel free to use suggestions for stuff like this, to streamline the review process for everyone :)

Reply:
Ack